README.md

```diff
@@ -55,3 +62,15 @@ try {
 
 ```
 
+## Limitations
+
+- Non-ember npm dependencies are not supported.  It is assumed all resolutions one of:
+   - classic ember apps
+   - classic ember addons
+   - v2 ember addons
+
+- No support for app namespace merging: so files in an addon's `app` directory will not be resolved.
+
+- No support for path conflict resolution.  This is unlikely to affect anybody, but if you have an Ember app with `my-app/app/tests/foo`, `my-app/tests/foo` will not resolve to this file, but instead only look for `my-app/tests/foo`
```

Comment:
This is "easy" to add support for, but would require more I/O so should be behind a flag that opts-in to spend I/O for this feature

Reply:
Can we do a one time "test" for this and print a warning?

Reply:
@rwjblue I think so, but not quite following the suggestion.

Are you saying that the first time you ask for `tests/` for something we determine is a classic app, check to see if `${myApp}/app/tests` exists and print a warning?

Reply:
cached per app in-process or something

Reply:
per-process module-scoped cached seems fine.